### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by rambudha**
ty

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,6 +2,5 @@ module.exports = function tiny(string) {
   if (typeof string !== "string") throw new TypeError("Tiny wants a string!");
   return string.replace(/\s/g, "");
 };
-/*This is just another branch of the space remover
-for testing purpose
-*/
+
+//typing
```
Comment (rawatnaresh) [inline index.js:6]: fjdkfjkdf

Comment (rambudha) [reply index.js:6]: done

**Review by rambudha — COMMENTED** [review]
onee

File: index.js (hunk 1 above)
Comment (rawatnaresh) [reply index.js:6]: okay

**Comment by rambudha** [comment]
closed

**Pull request closed without merging**